src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1454,6 +1461,7 @@
 
   <Node Name="BoundConstantPattern" Base="BoundPattern">
     <Field Name="Value" Type="BoundExpression"/>
+    <Field Name="ConstantValue" Type="ConstantValue" Null="allow"/>
```

Comment:
Why cannot we use ConstantValue of the Value field instead?

---
In reply to: [64666300](https://github.com/dotnet/roslyn/pull/11485#discussion_r64666300) [](ancestors = 64666300)